codewof/programming/content/en/rollercoaster-ride/question.md

```diff
@@ -0,0 +1,16 @@
+# Rollercoaster ride
+
+Write a function `ride_the_rollercoaster(age, with_an_adult)` that takes a number and a boolean value (i.e. `True` or ` False`)
+and **prints** a message saying whether you can ride the rollercoaster.
+
+To ride the rollercoaster a person needs to be 10 or older,
+or if they are younger they need to be accompanied by an adult.
+
+If the person can ride the rollercoaster then the function should **print**
+"You can ride the rollercoaster".
```

Comment:
From memory it seems consistent for printed statements to be in backticks, is that right?